assets/scss/page-colour.scss

```diff
@@ -0,0 +1,45 @@
+@each $name, $colour in $page-colours {
```

Comment:
can this whole file be copied into the pre-existing assets/scss/color-mixer file please? Keeps things consistent and cleaner